### PR TITLE
Phase 1.2.0 merge to sprint-7

### PR DESCRIPTION
**Pull request by tarento-dharmateja**

### CODE REVIEW

File: hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/function/BaseDispatcherFunction.scala [hunk 1]
```diff
@@ -93,8 +93,14 @@ abstract class BaseDispatcherFunction (config: BaseJobConfig)
         val payload = getPayload(payloadRefId);
         val payloadJSON = JSONUtil.serialize(payload);
         val result = DispatcherUtil.dispatch(recipientCtx, payloadJSON)
+        Console.println("result::"+result)
         //Adding updatedTimestamp for auditing
         event.put(Constants.UPDATED_TIME, Calendar.getInstance().getTime())
+<<<<<<< HEAD
```
Comment (prasadmoka) [inline hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/function/BaseDispatcherFunction.scala:99]: Remove this conflict issue

**Review by prasadmoka — CHANGES_REQUESTED** [review]
remove the merge conflicts

File: hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/function/BaseDispatcherFunction.scala (hunk 1 above)
Comment (tarento-dharmateja) [reply hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/function/BaseDispatcherFunction.scala:99]: sure

File: hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/function/BaseDispatcherFunction.scala [hunk 2]
```diff
@@ -93,8 +93,11 @@ abstract class BaseDispatcherFunction (config: BaseJobConfig)
         val payload = getPayload(payloadRefId);
         val payloadJSON = JSONUtil.serialize(payload);
         val result = DispatcherUtil.dispatch(recipientCtx, payloadJSON)
+        Console.println("result::"+result)
```
Comment (prasadmoka) [inline hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/function/BaseDispatcherFunction.scala:96]: remove this console.println and add logger

File: hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/function/BaseDispatcherFunction.scala [hunk 3]
```diff
@@ -93,8 +93,11 @@ abstract class BaseDispatcherFunction (config: BaseJobConfig)
...
         //Adding updatedTimestamp for auditing
         event.put(Constants.UPDATED_TIME, Calendar.getInstance().getTime())
+        Console.println("event::"+event)
```
Comment (prasadmoka) [inline hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/function/BaseDispatcherFunction.scala:99]: remove this console.println and add logger

**Review by prasadmoka — CHANGES_REQUESTED** [review]

**Review by prasadmoka — APPROVED** [review]

**Pull request merged**